tests/kafkatest/services/performance/consumer_performance.py

```diff
@@ -110,7 +110,7 @@ def args(self, version):
         if self.new_consumer:
             if version <= LATEST_0_10_0:
                 args['new-consumer'] = ""
-            args['broker-list'] = self.kafka.bootstrap_servers(self.security_config.security_protocol)
+            args['bootstrap-server'] = self.kafka.bootstrap_servers(self.security_config.security_protocol)
```

Comment:
Hmm.. The version check above suggests this tool still works with older versions. Not sure we'll be able to change these.

Reply:
Good catch